test/integration/jsonFilterTest.js

```diff
@@ -0,0 +1,27 @@
+const chai = require('chai');
+const expect = chai.expect;
+const ToughCookie = require('tough-cookie');
+
+const apiSettings = require('./api-settings.js');
+const { buildClient } = require('./client.js');
+
+const apiUrl = apiSettings.APIServerURL;
+const cookieJar = new ToughCookie.CookieJar();
+const api = buildClient(apiUrl, cookieJar);
+
+suite('Method test', async () =>
+{
+  test('Check Sort', async () => {
```

Comment:
I'm not sure this test is doing what it set out to do. It's validating that **some data is returned**, but it should be checking that the **right data is returned**.